makerootfs.sh

```diff
@@ -3,6 +3,9 @@
 #
 #     ./makerootfs.sh <image.yml> <fs> <output.img>
 #
+# The following env variables change the behaviour of this script
+#     ZEN_DEFAULT_BOOT - sets the default GRUB menu entry
```

Comment:
Add comment that ZEN_DEFAULT_BOOT=1 means using the qemu-friendly boot?

Reply:
Great point! I've added the missing comments